source/integrations/whatsapp.rst

```diff
@@ -0,0 +1,138 @@
+           """
+           Args:
+               name (str): The name of the server.
+               app_path (str): The path of the MindMeld application.
+               nlp (NaturalLanguageProcessor): MindMeld NLP component, will try to load from app path
+                 if None.
+           """
+           self.app = Flask(name)
+           if not nlp:
+               self.nlp = NaturalLanguageProcessor(app_path)
+               self.nlp.load()
+           else:
+               self.nlp = nlp
+           self.conv = Conversation(nlp=self.nlp, app_path=app_path)
+           self.logger = logging.getLogger(__name__)
+
+           @self.app.route("/", methods=["POST"])
+           def handle_message():  # pylint: disable=unused-variable
+               incoming_msg = request.values.get('Body', '').lower()
+               resp = MessagingResponse()
+               msg = resp.message()
+               response_text = self.conv.say(incoming_msg)[0]
+               msg.body(response_text)
+               return str(resp)
+
+       def run(self, host="localhost", port=7150):
+           self.app.run(host=host, port=port)
+
+
+   if __name__ == '__main__':
+       app = Flask(__name__)
+       configure_logs()
+       server = WhatsappBotServer(name='whatsapp', app_path='.')
+       port_number = 5000
+       print('Running server on port {}...'.format(port_number))
+       server.run(host='localhost', port=port_number)
+
+You can download the human resource blueprint for an example implementation under `whatsapp_bot_server.py`.
+
+.. code:: console
+
+   mindmeld blueprint hr_assistant
+
+
+3. Start the human resource server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set the following environment variables and start the flask server.
+
+.. code:: console
+
+   python -m hr_assistant build
+   cd hr_assistant
+   python webex_bot_server.py
+
+
+3. Test the integration
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Start the ngrok channel. You can download the ngrok application from the Internet and then unzip it in a safe location.
+
+.. code:: console
+   ./ngrok http 5000
+
+   Session Status                online
+   Session Expires               7 hours, 59 minutes
+   Update                        update available (version 2.3.35, Ctrl-U to update)
+   Version                       2.3.29
+   Region                        United States (us)
+   Web Interface                 http://127.0.0.1:4041
+   Forwarding                    http://be84be34.ngrok.io -> http://localhost:5000
+   Forwarding                    https://be84be34.ngrok.io -> http://localhost:5000
+
+   Connections                   ttl     opn     rt1     rt5     p50     p90
+                                 0       0       0.00    0.00    0.00    0.00
+
+After running ngrok application, copy the ngrok URL and paste into Twilio sandbox's configuration. Now you can converse with HR assistant on WhatsApp!
+
+.. image:: /images/whatsapp_sandbox.png
+    :width: 700px
+    :align: center
```

Comment:
After this image, add "Now you can converse with HR assistant on WhatsApp." And following that another image which is a screenshot of some example interactions with the Sandbox bot on web.whatsapp.com.